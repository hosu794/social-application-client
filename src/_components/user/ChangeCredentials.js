import React from "react";

import { Formik, Field, Form, ErrorMessage } from "formik";
import * as Yup from "yup";
import axios from "axios";
import { useDispatch, useSelector } from "react-redux";
import { userActions, authActions } from "../../_actions";

import FileUpload from "./FileUpload";
import ChangePassword from "./ChangePassword";
import ChangeUsername from "./ChangeUsername";

function ChangeCredentials() {
  const dispatch = useDispatch();

  return (
    <div class="container has-text-centered columns">
      <ChangeUsername />
      <ChangePassword />
      <FileUpload />
    </div>
  );
}

export default ChangeCredentials;
