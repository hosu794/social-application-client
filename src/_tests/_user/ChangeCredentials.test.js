import React from "react";
import { shallow } from "enzyme";
import ChangeCredentials from "../../_components/user/ChangeCredentials";

describe("Test for the ChangeCredentials Component", () => {
  let wrapper = shallow(<ChangeCredentials />);
  test("should render ChangeCredentials correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("should ChangeCredentials Component to be defined", () => {
    expect(wrapper).toBeDefined();
  });
});
