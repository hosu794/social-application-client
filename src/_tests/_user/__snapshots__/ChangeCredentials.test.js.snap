// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the ChangeCredentials Component should render ChangeCredentials correctly 1`] = `ShallowWrapper {}`;
